.github/workflows/create-tarball.yml

```diff
@@ -2,6 +2,22 @@ name: Create tarball including submodules
 
 on: [workflow_dispatch]
 
+permissions:
+  actions: read
+  checks: none
+  contents: write
+  deployments: none
+  id-token: none
+  issues: none
+  discussions: none
+  packages: none
+  pages: none
+  pull-requests: none
+  repository-projects: none
+  security-events: none
+  statuses: none
```

Comment:
This could have been done with a `default: none` action, but this is fine. Thanks!

Reply:
You'r welcome. I though it is a good idea to explicitely state what is not allowed with this token.

Reply:
The advantage of `default:` is moving forward: if GitHub adds more capabilities for the token, `default:none` will assure that they are not enabled here. Also, in general, fewer lines are easier to audit than more lines. 

However, there's another principle at work, which is _if is isn't broke, don't fix it_. Since what you committed works, I'm going to leave it!